hugegraph-api/src/main/java/com/baidu/hugegraph/auth/StandardAuthenticator.java

```diff
@@ -95,7 +101,12 @@ public void setup(HugeConfig config) {
         String graphName = config.get(ServerOptions.AUTH_GRAPH_STORE);
         String graphPath = config.getMap(ServerOptions.GRAPHS).get(graphName);
         E.checkArgument(graphPath != null,
-                        "Invalid graph name '%s'", graphName);
+                        "Can't find graph name '%s' in config '%s' at " +
+                        "'rest-server.properties' to store auth information, " +
+                        "please ensure the value of '%s' matches it correctly",
+                        graphName, ServerOptions.GRAPHS,
```

Comment:
ServerOptions.GRAPHS.name()?

Reply:
GRAPHS.name() only shows `graphs` in error (not clear enough)

GRAPHS.toString() returns name + value (seems more friendly for user to find out mismatch)

Reply:
ok